src/js/popper.js

```diff
@@ -0,0 +1,1210 @@
+          fn: function (e, t) {
+            var o = t.boundariesElement || p(e.instance.popper);
+            e.instance.reference === o && (o = p(o));
+            var n = B("transform"),
+              i = e.instance.popper.style,
+              r = i.top,
+              s = i.left,
+              d = i[n];
+            (i.top = ""), (i.left = ""), (i[n] = "");
+            var a = v(
+              e.instance.popper,
+              e.instance.reference,
+              t.padding,
+              o,
+              e.positionFixed
+            );
+            (i.top = r), (i.left = s), (i[n] = d), (t.boundaries = a);
+            var l = t.priority,
+              f = e.offsets.popper,
+              m = {
+                primary: function (e) {
+                  var o = f[e];
+                  return (
+                    f[e] < a[e] &&
+                      !t.escapeWithReference &&
+                      (o = ee(f[e], a[e])),
+                    ae({}, e, o)
+                  );
+                },
+                secondary: function (e) {
+                  var o = "right" === e ? "left" : "top",
+                    n = f[o];
+                  return (
+                    f[e] > a[e] &&
+                      !t.escapeWithReference &&
+                      (n = Q(
+                        f[o],
+                        a[e] - ("right" === e ? f.width : f.height)
+                      )),
+                    ae({}, o, n)
+                  );
+                },
+              };
+            return (
+              l.forEach(function (e) {
+                var t =
+                  -1 === ["left", "top"].indexOf(e) ? "secondary" : "primary";
+                f = le({}, f, m[t](e));
+              }),
+              (e.offsets.popper = f),
+              e
+            );
+          },
+          priority: ["left", "right", "top", "bottom"],
+          padding: 5,
+          boundariesElement: "scrollParent",
+        },
+        keepTogether: {
+          order: 400,
+          enabled: !0,
+          fn: function (e) {
+            var t = e.offsets,
+              o = t.popper,
+              n = t.reference,
+              i = e.placement.split("-")[0],
+              r = Z,
+              p = -1 !== ["top", "bottom"].indexOf(i),
+              s = p ? "right" : "bottom",
+              d = p ? "left" : "top",
+              a = p ? "width" : "height";
+            return (
+              o[s] < r(n[d]) && (e.offsets.popper[d] = r(n[d]) - o[a]),
+              o[d] > r(n[s]) && (e.offsets.popper[d] = r(n[s])),
+              e
+            );
+          },
+        },
+        arrow: {
+          order: 500,
+          enabled: !0,
+          fn: function (e, o) {
+            var n;
+            if (!K(e.instance.modifiers, "arrow", "keepTogether")) return e;
+            var i = o.element;
+            if ("string" == typeof i) {
+              if (((i = e.instance.popper.querySelector(i)), !i)) return e;
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)